__tests__/integration.test.js

```diff
@@ -94,3 +94,45 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("responds with a 200 status code", () => {
+    return request(app).get("/api/articles").expect(200);
+  });
+  test("200: responds with an articles array of objects with the correct properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        body.articles.forEach((property) => {
+          expect(typeof property.article_id).toBe("number");
+          expect(typeof property.title).toBe("string");
+          expect(typeof property.topic).toBe("string");
+          expect(typeof property.author).toBe("string");
+          expect(typeof property.created_at).toBe("string");
+          expect(typeof property.votes).toBe("number");
+          expect(typeof property.article_img_url).toBe("string");
+          expect(typeof property.comment_count).toBe("string");
+        });
+      });
+  });
+  test("200: should respond with an array of articles without a body property present on any of the objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        body.articles.forEach((property) => {
+          expect(property.body).toBeUndefined();
+        });
+      });
+  });
+  test.todo(
```

Comment:
If you were unsure of what to test this should have been an NCHelp not a pull request for review. You should be using jest sorted to test that your objects are returned in the correct order. The format is a correct utc date format which can be read by that package without any data manipulation.